docs/docs/ref-02-component-api.md

```diff
@@ -66,11 +66,11 @@ Like `setState()` but deletes any pre-existing state keys that are not in nextSt
 forceUpdate([function callback])
 ```
 
-If your `render()` method reads from something other than `this.props` or `this.state`, you'll need to tell React when it needs to re-run `render()` by calling `forceUpdate()`. You'll also need to call `forceUpdate()` if you mutate `this.state` directly.
+By default, updating your component's `props` or `state` will cause your component to re-render. However, if these change implicitly (eg: data deep within an object changes without changing the object itself) or if your `render()` method depends on some other data, you can tell React that it needs to re-run `render()` by calling `forceUpdate()`.
```

Comment:
Sounds good. The only part about this that I'm not wild about is "updating your component's props", which can be read to imply that you can set `this.props.foo`. It's almost not worth mentioning `props` at all for `forceUpdate`, we've seen very few issues as a result of props being the same reference (and that really would require further explanation). Overwhelmingly the issue comes up with modifying `this.state` directly so we should probably focus on that (and keep the "other data bit too").